### ISSUE TITLE
Undefined operation power

### ISSUE THREAD

**Issue by Jtiem** [modularscale/modularscale-sass#58]
Undefined operation: "power(minor-sixth(), 1) times 16px".)

I am going around and around trying to fix this error. I have been able to get past it before some how but for the life of me I cannot now.

I have installed gems for modular-scale and sassy-math and configured the config.rb file to require modular-scale.

Please help.

**Comment by rpkoller** [comment]
I ran into a similar issue:

 error style.scss (Line 131 of _vertical_rhythm.scss: Undefined operation: "2 times ms(0)".)

Is it maybe a Compass issue that plugins even they are properly imported aren't recognized anymore?

**Comment by scottkellum** [comment]
Modular Scale 2.0 will fix this. You can install it with `gem install modular-scale --pre` but I still need to update docs

**Comment by rpkoller** [comment]
Cool version 2.0 is ready and only updated docs missing. The only thing what confuses me why modular scale 1.x doesn't show that behaviour in every project.Happened only in one so far. Will give the pre version a try later today. Thanks!

**Comment by scottkellum** [comment]
@rpkoller sounds like a dependency issue with Sassy Math.

**Comment by rpkoller** [comment]
@scottkellum i've updated to the prerelease version. Before only one project failed. Now all projects fail preprocessing. :/ 

The odd project i've mentioned before sticks to the error message:

```
error style.scss (Line 131 of _vertical_rhythm.scss: Undefined operation: "2 times ms(0)".)
```

one other projects which worked under 1.x before throws the following error: 

```
error style.scss (Line 10 of _cache.scss: Invalid CSS after "...es} #{$Ratios}"": expected ")", was ": (")
```

Hm and Sassy Math is installed. The strange part is it worked for weeks without a problem and from one day to another things turned nasty. ;) I am unable to figure out a common pattern which might have lead to it.

**Comment by scottkellum** [comment]
@rpkoller Just pushed an update 2.0.0.alpha2 that might help.

Also ratios are stored as variables now `$golden` not functions `golden()` in this version. I think that might be the only big user facing change.

**Comment by rpkoller** [comment]
@scottkellum thanks for the quick followup! I will change the ratios to variables now. And a bit of an embarrassing question. How do you update prerelease gems? i thought it would be shown under gem outdated, but it is not? Do i just have to retype gem install modular-scale --pre ? Best regards Ralf

**Comment by scottkellum** [comment]
@rpkoller Yeah, `gem install modular-scale --pre` should do it for you. You may have to `gem uninstall modular-scale` to uninstall previous versions. (I’m not super well versed in this stuff myself).

**Comment by rpkoller** [comment]
@scottkellum sorry for the delayed answer but i had no access to my computer the last week and had to leave in more or less of a hurry so i had no Github login credentials on my comp nor it would have helped since i would have been unable to test things. 

Did that now with Alpha 2 finally but i get:

```
error style.scss (Line 4 of _modular-scale.scss: Undefined variable: "$major-third".)
```

It is error'ing more or less on the first occurrence of a scale variable: 

```
$base-size: 20px 44px;
$ratio: $major-third $fifth;
```

The whole thing seems somehow buggered. But Modular Scale is required and imported properly before. Real strange. :/ 

Shall i create a separate issue in the tracker here?

**Comment by scottkellum** [comment]
@rpkoller can you type `gem list modular-scale` in your terminal and let me know what it returns?

**Comment by rpkoller** [comment]
@scottkellum it returns (i've uninstalled alpha1 first before i've intalled alpha2 like you recommended)

```
*** LOCAL GEMS ***

modular-scale (2.0.0.alpha2)
```

**Comment by scottkellum** [comment]
@rpkoller Lets discuss over at #63.

**Comment by scottkellum** [comment]
[Should be fixed in 2.0](https://github.com/Team-Sass/modular-scale/blob/2.x/readme.md)
